src/main/java/com/github/tomakehurst/wiremock/http/ContentTypeHeader.java

```diff
@@ -17,21 +17,25 @@
 
 import com.github.tomakehurst.wiremock.common.Strings;
 import java.nio.charset.Charset;
+import java.util.Map;
 import java.util.Optional;
+import me.jvt.http.mediatype.MediaType;
 
 public class ContentTypeHeader extends HttpHeader {
 
   public static final String KEY = "Content-Type";
+  private static final String CHARSET_QUOTES = "\"([^\"]*)\"";
 
-  private String[] parts;
+  private final Optional<MediaType> maybeMediaType;
```

Comment:
This has an IDE warning, but I believe it's the right thing to do to give us the ability to more easily handle the absense of the value. Thoughts? (I've not done Java in ~18 months so my knowledge is rusty!)